### DISCUSSION TITLE
No error message is given for discussion category limit

### DISCUSSION THREAD

**Discussion by seanpm2001** [community/community#60075] [Discussions]
### Select Topic Area

Bug

### Body

I have been working on a project utilizing discussions. A few days ago, I tried to create a 26th discussion category. It acted like it was going to go through, then the button turned back to the unpressed position, meaning it didn't. I tried multiple times, and even waited until the next day, but still nothing. So I went to look up what was happening, and I found that there was a 25 discussion category limit.

My proposal is that there should be an error popup when you try to create a 26th category, saying something like "Could not create new category: You have reached the limit of 25 discussion categories" it would have helped me solve this problem a lot faster. It is really confusing when something is going wrong, but no feedback is given.

**Comment by dakshithadissanayaka** [comment]
I understand your frustration with the lack of error feedback when trying to create a 26th discussion category. It's certainly confusing and time-consuming to encounter an unresponsive action without any clear explanation.

Your proposal to implement an error popup is a sensible and user-friendly approach to address this issue. Providing a clear error message indicating the limit of 25 discussion categories would immediately inform the user of the restriction and prevent them from wasting time on repeated attempts.

Here's a breakdown of the benefits of implementing an error popup:

• **Improved User Experience:** The error popup would provide timely and relevant feedback, saving users from unnecessary effort and frustration.

• **Reduced Confusion:** The clear explanation would eliminate ambiguity and ensure users understand the reason for the failed action.

• **Enhanced Usability:** The error message would guide users towards an alternative solution, preventing them from getting stuck in an endless loop of attempts.

Overall, implementing an error popup for exceeding the discussion category limit would significantly improve the user experience and make the platform more intuitive and user-friendly.
